app/Fission/Internal/Development.hs

```diff
@@ -31,14 +39,17 @@ import           Fission.App (runApp)
      > runOne Network.IPFS.Peer.all
      > -- Right ["/ip4/3.215.160.238/tcp/4001/ipfs/QmVLEz2SxoNiFnuyLpbXsH6SvjPTrHNMU88vCQZyhgBzgw"]
 -}
-runOne :: RIO (Config PG) a -> IO a
+runOne :: RIO Config a -> IO a
 runOne action = do
-  logOptions   <- logOptionsHandle stdout True
+  logOptions <- logOptionsHandle stdout True
+  processCtx  <- mkDefaultProcessContext
+  httpManager <- HTTP.newManager HTTP.defaultManagerSettings
+
   withLogFunc (setLogUseTime True logOptions) \logFunc -> do
-    dbPool      <- runApp <| connPool 1 1 3600 pgConnectInfo
-    processCtx  <- mkDefaultProcessContext
-    httpManager <- HTTP.newManager HTTP.defaultManagerSettings
-    run logFunc dbPool processCtx httpManager action
+    withDBPool logFunc connectionInfo 4 \dbPool ->
```

Comment:
💅 Not sure what `4` is in this context. Could be useful if we move it to a named variable.